doc/manual/rl-next/nix-store-add.md

```diff
@@ -0,0 +1,7 @@
+---
+synopsis: Give `nix store add` a `--hash-algo` flag
+prs: 9809
+---
+
+Adds a missing feature that was present in the old CLI, and matches our
+plans to have similar flags for `nix hash convert` and `hash hash path`.
```

Comment:
nit:

```suggestion
plans to have similar flags for `nix hash convert` and `nix hash path`.
```

Reply:
Oh, I see this was from #9809. Once you rebase, maybe you could fix this typo up regardless? :smile:

Reply:
Done in #9821